windows-agent/internal/proservices/landscape/executor.go

```diff
@@ -348,9 +348,14 @@ func wantRootfsChecksum(u *url.URL) (string, error) {
 	checksumsURL := u.ResolveReference(shasRelativeURL)
 
 	resp, err := http.Get(checksumsURL.String())
+	// Errors here are protocol errors, not 404s. "A non-2xx response doesn't cause an error".
 	if err != nil {
 		return "", fmt.Errorf("could not download checksums file %q: %v", checksumsURL, err)
 	}
+	if resp.StatusCode == http.StatusNotFound {
+		log.Warningf(ctx, "expected checksums file %q not found", checksumsURL)
```

Comment:
I would degrade that to an Infof and replace `expected` with nothing.

(if it was really expceted, that would be an error and we said we won’t enforce it)